### ISSUE TITLE
Installers Lack of Accessibility with orca

### ISSUE THREAD

**Issue by Joel8601** [canonical/ubuntu-desktop-provision#896]
### Is there an existing issue for this?

- [X] I have searched the existing issues and found none matching what I'm reporting.

### Bug Description

The Ubuntu-mate installers widgets are not fully accessible by the screen readers which means only some elevated buttons are only read by the screen reader, so it makes the people with visual impairment almost impossible to install Ubuntu without another persons help.I think this something related to semantics where the label or hints for the screen reader are not defined.Please Look into it as soon as possible and hope it gets fixed soon.

### Steps to reproduce the behavior

1.Open the installers wizard
2.Start the screen reader 
3.Try to navigate through the installer only with the screen reader(keep your eyes closed) 

### Ubuntu release

24.04 LTS

### What architecture are you using?

amd64

### System info

6.8.0-51-generic #52-Ubuntu SMP PREEMPT_DYNAMIC Thu Dec  5 13:09:44 UTC 2024
name:      ubuntu-desktop-bootstrap
summary:   Ubuntu Desktop Bootstrap
publisher: Canonical✓
store-url: https://snapcraft.io/ubuntu-desktop-bootstrap
license:   GPL-3.0
description: |
  This project is a modern implementation of the Ubuntu Desktop installer,
  using subiquity as a backend and Flutter for the UI.
snap-id: dLfoSWlQziHta7kJaco3IhnPwGVJ3bIt
channels:
  latest/stable:    0+git.66bf8d183 2024-07-24 (237) 122MB classic
  latest/candidate: 0+git.2f88d9b70 2024-08-19 (242) 122MB classic
  latest/beta:      ↑                                      
  latest/edge:      ↑                                      
  25.04/stable:     0+git.739a47078 2024-12-04 (297) 110MB classic
  25.04/candidate:  ↑                                      
  25.04/beta:       ↑                                      
  25.04/edge:       0+git.ded09ee4d 2024-12-10 (299) 110MB classic
  24.10/stable:     0+git.8f7b50565 2024-10-08 (290) 111MB classic
  24.10/candidate:  ↑                                      
  24.10/beta:       ↑                                      
  24.10/edge:       0+git.8f7b50565 2024-10-08 (290) 111MB classic
  24.04/stable:     0+git.66bf8d183 2024-08-14 (237) 122MB classic
  24.04/candidate:  ↑                                      
  24.04/beta:       ↑                                      
  24.04/edge:       0+git.77516091c 2024-09-25 (275) 117MB classic
---
name:      snapd
summary:   Daemon and tooling that enable snap packages
publisher: Canonical✓
store-url: https://snapcraft.io/snapd
contact:   https://github.com/snapcore/snapd/issues
license:   GPL-3.0
description: |
  Install, configure, refresh and remove snap packages. Snaps are
  'universal' packages that work across many different Linux systems,
  enabling secure distribution of the latest apps and utilities for
  cloud, servers, desktops and the internet of things.
  
  Start with 'snap list' to see installed snaps.
type:         snapd
snap-id:      PMrrV4ml8uWuEUDBT8dSGnKUYbevVhc4
tracking:     latest/stable
refresh-date: 22 days ago, at 16:13 IST
channels:
  latest/stable:    2.66.1               2024-11-27 (23258) 46MB -
  latest/candidate: 2.67                 2024-12-17 (23545) 46MB -
  latest/beta:      2.67                 2024-12-04 (23545) 46MB -
  latest/edge:      2.67+git101.g1f677f1 2024-12-19 (23636) 50MB -
installed:          2.66.1                          (23258) 46MB snapd

### Additional context

I tried implementing semantics but cant find a way to run my code, if you could also guide me on where to implement semantics and on how to execute the code after making changes and how to test and deploy it will be a great help as i am customising Ubuntu-mate so itwould me more helpful
